BlueprintUICommonControls/Tests/Sources/XCTestCaseExtensions.swift

```diff
@@ -104,6 +104,8 @@ extension XCTestCase {
             view.frame = CGRect(origin: .zero, size: size)
         } else {
             view.sizeToFit()
+            view.frame.size.width.round(.up, by: scale)
+            view.frame.size.height.round(.up, by: scale)
```

Comment:
This allows snapshots to continue working in 1x scale even when the underlying elements are using the screen scale to measure.